cms/settings.py

```
@@ -523,11 +520,41 @@
 MFA_EXCLUDE_PATHS = ['/fu/', '/api/', '/manage/', '/accounts/']
 
 WHISPER_CPP_DIR, WHISPER_CPP_COMMAND, WHISPER_CPP_MODEL = get_whisper_cpp_paths()
-from .local_settings import *
 ALLOWED_HOSTS.append(FRONTEND_HOST.replace("http://", "").replace("https://", ""))
 WHISPER_SIZE = "base"
 
+from .local_settings import *
```

Ruff (0.12.2)

`from .local_settings import *` used; unable to detect undefined names

(F403)